Jenkinsfile

```diff
@@ -14,7 +14,7 @@ pipeline {
           Restrict weekly builds to master branch, all others will be built on change only.
           Note: The BRANCH_NAME will only work with a multi-branch job using the github-branch-source
         */
-        cron(BRANCH_NAME == "master" ? "@weekly" : "")
+        pollSCM(BRANCH_NAME == "master" ? "H/15 * * * 1-5" : "")
```

Comment:
✏️ As rarely as this repo changes, every 15 minutes might be overkill. Maybe hourly or daily?

Reply:
changed to @daily per our agreement